.gitignore

```diff
@@ -12,7 +12,8 @@
 */light-test-cache/
 /**/.DS_store
 /cli/bin
-
+*/light-anchor 
+light-anchor
```

Comment:
Why do we need this? Install.sh script installs light-anchor into .local/, which is already in gitignore

Reply:
good catch this is not needed